shared/admin-tools/user/admin.js

```diff
@@ -76,6 +76,56 @@ const deactivateAdminAccount = async () => {
     .promise();
 };
 
+/**
+ * This verifies that the USTC admin user is disabled in Cognito
+ */
+const verifyAdminUserDisabled = async ({ attempt }) => {
+  const cognito = new CognitoIdentityServiceProvider({ region: 'us-east-1' });
```

Comment:
nit-pick: Possible use a constant for `us-east-1`

Reply:
Unfortunately, I think this is an artifact with github.  I did not make this change.